Replaced slang terms (#8719)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/about-exchange-online-admin-role.md

- MOE150
- GEA150
ms.assetid: 097ae285-c4af-4319-9770-e2559d66e4c8
description: "Exchange administrators manage your organization's email and mailboxes and, for example, recover deleted items in a user's mailbox."
---

# About the Exchange Administrator role

To help you administer Microsoft 365, you can [assign](assign-admin-roles.md) users permissions to manage your organization's email and mailboxes from the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center</a>. You do this by assigning them to the Exchange Administrator role.
  

>  **Tip**: When you assign someone to the Exchange Administrator role, it is recommended to assign them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.

Reviewer: ```suggestion
> [!TIP]
> When you assign someone to the Exchange Administrator role, we recommend assigning them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
```

> description: "Exchange Online admins manage your organization email and mailboxes and, for example, recover deleted items in a user's mailbox."

Reviewer: ```suggestion
description: "Exchange administrators manage your organization's email and mailboxes and, for example, recover deleted items in a user's mailbox."
```

---

# About the Exchange Administrator role

To help you administer Microsoft 365, you can [assign](assign-admin-roles.md) users permissions to manage your organization's email and mailboxes from the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center</a>. You do this by assigning them to the Exchange Administrator role.
  

>  **Tip**: When you assign someone to the Exchange Administrator role, it is recommended to assign them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.

Reviewer: ```suggestion
 **Tip**: When you assign someone to the Exchange Administrator role, we recommend assigning them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
```
